src/node_jsvmapi.h

```diff
@@ -288,13 +321,17 @@ NODE_EXTERN bool napi_is_exception_pending(napi_env e);
 NODE_EXTERN napi_value napi_get_and_clear_last_exception(napi_env e);
```

Comment:
Above methods (for handle scopes and exceptions) still need to be updated to return `napi_status`. You were probably aware, but I'm flagging this since it's the only remaining change needed in the public API.